src/graphql/query/userAchievementsProgress.graphql

```diff
@@ -0,0 +1,11 @@
+
+query userAchievementProgress(
+	$userId: String!
+) {
+	userAchievementProgress(userId: $userId) {
```

Comment:
This endpoint should be refactored on the backend to read the user ID from the JWT, like other endpoints do. Then the userId argument isn't required and this can be called in a regular prefetch like other endpoints.